ChristopherPenaVelez/kmq
# Profile Page & Leaderboard Fix

Summary of Changes:

Added a “Profile” page (/profile) that shows the currently authenticated user’s scores.
Fixed the leaderboard route in score.ts by adding GET /api/score/top to retrieve the top scores globally.
Updated leaderboard/page.tsx to fetch GET /api/score/top (including cookies if needed).
Confirmed both pages now display correct data from the database.
How to Test:

Sign up or log in, then play the 5-round quiz to generate scores.
Check /profile to see your personal scores.
Check /leaderboard to see the top 5 global scores.
